Sources/Comet/Helpers/Logging/Entity/RequestLogLevel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  RequestLogLevel.swift
+//  Comet
+//
+//  Created by Jaroslav Novák on 17.01.2022.
+//  Copyright © 2022 Etnetera. All rights reserved.
+//
+
+import Foundation
+
+public struct RequestLogLevel: OptionSet {
+    public let rawValue: UInt8
+
+    public init(rawValue: UInt8) {
+        self.rawValue = rawValue
+    }
+
+    public static let none: RequestLogLevel = []
+    public static let url = RequestLogLevel(rawValue: 1)
+    public static let headers = RequestLogLevel(rawValue: 0b11)
```

Comment:
Is this right that the URL bit is always present?

Reply:
I would say that always showing url is right.